### PR TITLE
Address 44

### PR DESCRIPTION
**Pull request by lucasfth**
My fix was to just remove the stars.
The reasoning is that they can not be used to unstar the starred courses anyway.

image

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -207,6 +207,11 @@ html .lightModeOnly {
   visibility: hidden !important;
 }
 
+/* remove star sign on starred courses */
+.text-primary {
```
Comment (PatrickMatthiesen) [inline src/styles/main.css:211]: Its probably not a good idea to hide this selector as it is a default color selector.
image
As you can see then it has hidden a button somewhere

**Review by PatrickMatthiesen — CHANGES_REQUESTED** [review]
It might just be a better idea to remove or unset the position from the selector in main.css?
https://github.com/PhilipFlyvholm/learnit-plus-plus/blob/1bdfe5e35da62799c65c7c2421d79489646075a3/src/styles/main.css#L158-L162
image

**Comment by lucasfth** [comment]
I tried to unset the position, which resulted in a fairly inconsistent look:

image

I tried to move the star icon to the bottom right. But due to the reorganization of the texts I seem to be unable to make the bottom fill out more (view below):

image

If it could be done it should be possible to make the star icon fit to the bottom right of the parent

**Comment by PatrickMatthiesen** [comment]
Maybe it would just be an easier fix to set the width of the link/name of the course to 90-95%. That way we force the header to wrap before it gets to the star?
image
image

**Comment by PhilipFlyvholm** [comment]
If we instead just removed some CSS I added some time ago then it would be at the bottom and respect the titles:

See: https://github.com/PhilipFlyvholm/learnit-plus-plus/blob/1bdfe5e35da62799c65c7c2421d79489646075a3/src/styles/main.css#L158C2-L162C2

The position at the top was to try to match the old LearnIT, and because the original feature request wanted the order reversed like it is currently. Of course, the star part could have been implemented better.

**Comment by PhilipFlyvholm** [comment]
@lucasfth I have pushed a change that solves the same problem but with a different solution. I hope this is okay.

**Review by PatrickMatthiesen — APPROVED** [review]
Looks good to me :octocat:

**Pull request merged**